apps/pom.xml

```diff
@@ -175,27 +179,27 @@
 			<dependency>
 				<!-- Lock & harmonize the versions of the various Spring Framework modules. 
 					We look to HAPI for a guide on what release series to use: ca.uhn.hapi.fhir:hapi-fhir-base:pom:5.7.2
-					uses the 5.3.18 version of the Spring Framework. -->
+					uses the 5.3.20 version of the Spring Framework. -->
 				<groupId>org.springframework</groupId>
 				<artifactId>spring-framework-bom</artifactId>
-				<version>5.3.18</version>
+				<version>5.3.20</version>
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
 			<dependency>
 				<!-- This is the latest release compatible with the version of Spring 
-					Framework that we're using, per http://docs.spring.io/spring-data/jpa/docs/1.11.4.RELEASE/reference/html/#dependencies.spring-framework. -->
+					Framework that we're using, per https://docs.spring.io/spring-data/jpa/docs/2.7.1/reference/html/#dependencies.spring-framework. -->
```

Comment:
according to this link we need spring 5.3.21 or better but we have 5.3.20 (and we need to stick to 5.3.20 because that matches HAPI). So I think we need to look for a slightly earlier version of spring-data-jpa that is compatible with our spring verision

Reply:
Updated